<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisconnectionScreen</class>
 <widget class="QWidget" name="DisconnectionScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="disconnectionLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#disconnectionLabel {
    qproperty-alignment: AlignCenter;
    font-family: &quot;Muli SemiBold&quot;;
    font-size: 28px;
	color: #58595b;
	line-height:1px;
}</string>
   </property>
   <property name="text">
    <string>Şarj kablosunu araçtan çıkartıp 
cihazdaki yerine yerleştirin</string>
   </property>
  </widget>
  <widget class="QWidget" name="VehicleAndStation" native="true">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>104</y>
     <width>637</width>
     <height>294</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="infoLabelWidget" native="true">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>420</y>
     <width>425</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#energyLabel, #durationLabel, #batteryLabel, #energyValue, #durationValue, #batteryLevelValue {
    qproperty-alignment: AlignCenter;
    font-family: &quot;Muli SemiBold&quot;;
    font-size: 20px;
    color: #58595b;
    line-height:1px;
}

#energyLabel, #durationLabel, #batteryLabel {
    font-weight: bold;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string>Duration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="durationValue">
        <property name="text">
         <string>2 h</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="energyDurationSeparator">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="energyLabel">
        <property name="text">
         <string>Enerji</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="energyValue">
        <property name="text">
         <string>10 kWh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="batteryLabel">
        <property name="text">
         <string>Battery Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="batteryLevelValue">
        <property name="text">
         <string>%70</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <widget class="Line" name="durationBatterySeparator">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
